po/fr.po

```diff
@@ -0,0 +1,55 @@
+# gimp3-fourier-plugin
+# Copyright (C) 2024
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: gimp30-fourier-plugin\n"
+"Report-Msgid-Bugs-To: https://github.com/rpeyron/plugin-gimp-fourier/issues\n"
+"POT-Creation-Date: 2024-05-21 15:18-0700\n"
+"PO-Revision-Date: 2024-05-21 00:00+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../fourier.c:420
+msgid "_Fourier..."
+msgstr ""
```

Comment:
I guess we should not provide a po file with empty translations? I will provide the french translations.

Reply:
Yes. A simple text editor will work.
If you want to create other new locales, you need to edit the top of po/Makefile.am,
then copy gimp3-fourier.pot to ??.po and same process there too.
You can use gimp3-fix-ca/po/Makefile.am as example